src/Neo.VM/ExecutionEngine.cs

```diff
@@ -288,7 +288,7 @@ public T Pop<T>() where T : StackItem
         /// </summary>
         protected virtual void PostExecuteInstruction(Instruction instruction)
         {
-            if (ReferenceCounter.Count < Limits.MaxStackSize) return;
+            if (ReferenceCounter.Count <= Limits.MaxStackSize) return;
```

Comment:
please make this one a seperate pr.

Reply:
I am still considering it. If <= is not applied, this PR makes no sense.

Reply:
> I am still considering it. If <= is not applied, this PR makes no sense.

you can make <= merhged first, then do this pr. If you mixed together, will be slower to review.